mapie/tests/test_utils.py

```diff
@@ -410,3 +410,33 @@ def test_change_values_zero_one() -> None:
     """Test that binary output are changed to zero one outputs."""
     array_ = check_binary_zero_one(np.array([0, 4, 4]))
     assert (np.unique(array_) == np.array([0, 1])).all()
+
+
+@pytest.mark.parametrize("cv", [5, "split"])
+def test_check_cv_same_split_with_random_state(cv: BaseCrossValidator) -> None:
+    """Test that cv generate same split with fixed random_state."""
+    cv = check_cv(cv, random_state=random_state)
+
+    train_indices_1, train_indices_2 = [], []
+    for train_index, _ in cv.split(X):
+        train_indices_1.append(train_index)
+    for train_index, _ in cv.split(X):
+        train_indices_2.append(train_index)
+
+    for i in range(cv.get_n_splits()):
+        np.testing.assert_allclose(train_indices_1[i], train_indices_2[i])
+
+
+@pytest.mark.parametrize("cv", [5, "split"])
```

Comment:
I think we should also test for `cv=-1`.

Reply:
I will test it, thanks :)

Reply:
In fact, it is not possible because with `cv=-1`, we need to have `X` (and in this test, we don't have `X`).